boards/arm/nrf9160_pca20035/board_nonsecure.c

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(board_nonsecure, CONFIG_BOARD_LOG_LEVEL);
```

Comment:
Let's move this after the `#include` directives.

Reply:
Fixed